FitnesseRoot/HttpTestSuite/FileServerTestSuite/CreateReadUpdateDelete/content.txt

```diff
@@ -0,0 +1,21 @@
+|script  |http browser                          |
```

Comment:
I've added some more context in the PR comment – I should've added it before, sorry.
I wanted to change these tests so that `POST` creates a new file where it doesn't exist, and `PUT` updates a file where it already exists.

I could add a test that `PUT`'s would 405 on a file that doesn't exist to make the difference between `POST` and `PUT` explicit. (I think this would also be a way to move the 405 behaviour out of the auth test as you mentioned above).

I could also put the `/form` tests back in the route-based tests as they didn't seem to correspond to an actual `form` file in the public dir. Then we could still test `content-type: application/x-www-form-urlencoded` behaviour? (The `POST` requests in Fitnesse didn't have any `content-type` header before which I think was also pretty confusing)

Reply:
Good call on adding the `Content-Type` header. That's definitely an important thing that was missing.